### PR TITLE
Add TextArea component

### PR DESCRIPTION
**Pull request by kimak**
## Checklist before requesting a review
- [x] Working on iOS
- [x] Working on Android
- [x] Integration tests added
- [x] Visual regressions screenshots are up to date
- [x] Design validated

### CODE REVIEW

File: packages/sandbox/e2e/ios/screenshots/InputTextArea.png
Comment (ianrobertlambert) [inline packages/sandbox/e2e/ios/screenshots/InputTextArea.png:1]: Is it possible to see the `InputTextArea` displayed with `PrimitiveCaption` enabled?

File: packages/sandbox/e2e/android/screenshots/InputTextArea.png
Comment (ianrobertlambert) [inline packages/sandbox/e2e/android/screenshots/InputTextArea.png:1]: Is it possible to see the `InputTextArea` displayed with `PrimitiveCaption` enabled?

**Review by judastheo — APPROVED** [review]

File: packages/sandbox/e2e/android/screenshots/InputTextArea.png
Comment (kimak) [reply packages/sandbox/e2e/android/screenshots/InputTextArea.png:1]: done ✅

**Comment by ianrobertlambert** [comment]
[Board updated](https://www.notion.so/luko/b151fb81699949aca449d3dc74776e28?v=3b436e36543c4cc7900badc1b5a9f582&p=97c9a5da57784163b1cfe4e7de4048de&pm=s) but there is no feedback :) Thanks!

**Pull request merged**